### DISCUSSION TITLE
Transition from cloud to self-hosted

### DISCUSSION THREAD

**Discussion by dgaiero** [Budibase/budibase#9112] [General]
Hello,
I am interested in using the self-hosted version of Budibase, however I don't have the infrastructure setup yet. I'd like to start learning Budibase and building some apps, but is it possible to export them and import to a self-hosted install at a later point?

Thanks!

**Comment by melohagan** [comment]
It is! More on backups here: https://docs.budibase.com/docs/backups-1

**Reply by BennaceurHichem** [reply]
@melohagan  But doing a backup and self-host it needs a paid plan. Is there a way to do that without subscribing to the premium version?

The app is already exsiting
